tests/cypress/page-object/digitall.home.page.ts

```diff
@@ -15,9 +21,24 @@ class DigitallHomePage extends BasePage {
             .contains('div[role="row"]', page)
             .rightclick({ force: true })
             .should('have.class', 'context-menu-open')
-        this.getIframeBody().contains('span', 'Edit').click()
+        this.getIframeBody().find(this.elements.editSite).click()
         return editPage
     }
+
+    publishSite() {
+        this.getIframeBody()
+            .contains('div[role="row"]', 'Digitall')
+            .rightclick({ force: true })
+            .should('have.class', 'context-menu-open')
+        this.getIframeBody().find(this.elements.publishSiteinAllLang).click()
+        return workflowDashboard
+    }
+
+    flushCache(){
+        this.getIframeBody().find(this.elements.cacheButton).click().debug()
```

Comment:
May I ask the usage of "debug" here?

Reply:
I forgot to remove it - will be removed in the next commit